app/page/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<!-- Wire, Copyright (C) Wire Swiss GmbH -->
+<!-- Wire, Copyright (C) 2016 Wire Swiss GmbH -->
```

Comment:
This is a different case though. File creation is the moment of serving it to the user. Thus it should be 2017. It is not our preamble in the git.